packages/myst-execute/src/execute.ts

```diff
@@ -175,15 +175,15 @@ function isCellBlock(node: GenericNode): node is CodeBlock {
  * @param node block to test
  */
 function codeBlockRaisesException(node: CodeBlock) {
-  return !!node.data?.tags?.includes?.('raises-exception');
+  return !!node.data?.tags?.includes?.(NotebookCellTags.raisesException);
 }
 /**
  * Return true if the given code block should not be executed
  *
  * @param node block to test
  */
 function codeBlockSkipsExecution(node: CodeBlock) {
-  return !!node.data?.tags?.includes?.('skip-execution');
+  return !!node.data?.tags?.includes?.(NotebookCellTags.skipExecution);
```

Comment:
Much better!

Reply:
Small little improvements. I was in there looking to implement this and was surprised it was already done!